src/themes/eLife/styles.css

```diff
@@ -300,3 +310,93 @@ blockquote {
     line-height: 1.56522;
   }
 }
+
+:--cite-group {
+  list-style: none;
+  counter-reset: cite-counter;
+  margin: 0;
+  padding: 0;
+  font-family: var(--heading-font);
+
+  ol,
+  ul {
+    padding-left: 0;
+    list-style: none;
+  }
+
+  > li {
```

Comment:
Not 100%, but should this be `& > li {...` or can you omit the `&`?

Reply:
Yeah if it's nested, it'll compile to the same thing as `& > li {...` with `> li {...` (but if it's part of the name of another selector, like `&--selector-name` the `&` is required).